clusterloader2/testing/load/config.yaml

```diff
@@ -89,12 +118,12 @@ steps:
     Method: APIResponsivenessPrometheus
     Params:
       action: start
-  - Identifier: PodStartupLatency
+  - Identifier: CreatePhasePodStartupLatency
```

Comment:
In fact this is a good change that can be left without if

Reply:
Let's move the renaming to a separate PR (so this line and the one at the end of the file)

Reply:
Done.